src/dislocker.rb

```diff
@@ -11,10 +11,12 @@ class Dislocker < Formula
   version "0.4.0"
 
   depends_on "polarssl"
-  depends_on "osxfuse"
+  depends_on :osxfuse
 
   def install
+    system "mkdir -p #{prefix}/{bin,lib,share/man/man1}"
     system "make -C src"
     system "make -C src install DESTDIR=#{prefix}/"
+    system "cp #{prefix}/lib/libdislocker.0.dylib #{prefix}/bin/"
```

Comment:
It doesn't seem like the right place to copy the library to, as bin is for binaries. There should be a place working more suitable if the `/usr/local/Cellar/dislocker/0.4.0/lib/` directory doesn't fit. I'll try to find it today and come back to you.